### DISCUSSION TITLE
Arduino IDE 2 auto complete/ Quick suggest functionality

### DISCUSSION THREAD

**Discussion by Gideon-Jonkers** [SpenceKonde/DxCore#455] [Future development discussion]
Arduino IDE 2 provides an auto complete or quick suggestions as they call it. It works like any other IDE auto complete. I have found it very handy, especially with the teensy core to know what functions are available without a google search. It can sometimes make development easier and convenient.

However I have found the the functionality does not work with the DxCore unfortunately. Is there maybe something I am missing or have done wrong?

**Comment by SpenceKonde** [comment]
I have not yet installed a copy of Arduino IDE 2.x for development because reports about it had been so dismal at first, and I need to catch up on current issues. 

Do you have any information on how we are supposed to specify autocomplete, since apparently it;'s not done through keywords.txt like you'd expect (then it would work)

**Comment by Gideon-Jonkers** [comment]
It feels like I am the only one that has decided to download the new IDE a bit later in its development. I have been looking on their repository to try and find how it works or how it gets enabled. I could not find anything.

I have decided to try PlatformIO for DxCore instead.
